sdk/plugin/serve.go

```diff
@@ -81,12 +81,73 @@ func Serve(opts *ServeOpts) error {
 	return nil
 }
 
+// ServeMultiplex is a helper function used to serve a backend plugin. This
+// should be ran on the plugin's main process.
+func ServeMultiplex(opts *ServeOpts) error {
+	logger := opts.Logger
+	if logger == nil {
+		logger = log.New(&log.LoggerOptions{
+			Level:      log.Trace,
```

Comment:
Is trace a good default? Info is probably the more common default

Reply:
I agree with you. However, this has been the default for the Serve function since the begging of (Vault) time. I will need to verify if we can change to Info going forward.

Reply:
I did some verification and I think this is ok to set as Info.